src/OMSimulatorLib/System.h

```diff
@@ -108,6 +108,12 @@ namespace oms3
     virtual oms_status_enu_t terminate() = 0;
     virtual oms_status_enu_t stepUntil(double stopTime) = 0;
 
+    oms_status_enu_t getReal(const ComRef& cref, double& value) const;
+    oms_status_enu_t setReal(const ComRef& cref, double value);
+    oms_status_enu_t getReals(const std::vector<ComRef> &crefs, std::vector<double> &values) const;
+    oms_status_enu_t setReals(const std::vector<ComRef> &crefs, std::vector<double> values);
+    oms_status_enu_t setRealInputDerivatives(const ComRef &cref, int order, double value);
```

Comment:
@robbr48 I moved `getReal`/`setReal` to `class System` to implement it only once for all system kinds. I moved your methods as well, but I am not certain about `setRealInputDerivatives`. Is it for system inputs or component (FMU?) inputs? Or for both?

Reply:
I guess it needs to be propagated through the connections in the same way as setReal.